application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -46,6 +46,10 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
 
         Guild guild = event.getGuild();
         Message message = event.getMessage();
+        
+        if (!message.getChannel().getType().isThread()) {
+            message.createThreadChannel("Discussion for " + message.getAuthor().getName() + "'s suggestion").queue();
```

Comment:
Please put it into a method, such as `createDiscussionThread(...);`

Ideally the code would then look like:
```java
if (...) {
  return;
}

...

createDiscussionThread(...);
reactWith(...);
reactWith(...);
```

Reply:
done